File: java/model/Item.java
package model;

public class Item {
    private String nome;
    private boolean concluido;
    private int prioridade;

    public Item(String nome){
        this.nome = nome;
        this.concluido = false;
        this.prioridade = 0;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public boolean isConcluido() {
        return concluido;
    }

    public void setConcluido(boolean concluido) {
        this.concluido = concluido;
    }

    public int getPrioridade() {
        return prioridade;
    }

    public void setPrioridade(int prioridade) {
        this.prioridade = prioridade;
    }
}

File: php web/models/Item.php
<?php

class Item {
    private $nome;
    private $concluido;
    private $prioridade;

    public function __construct($nome) { 
        $this->nome = $nome;
        $this->concluido = false;
	    $this->prioridade = 0;
    }
    
    public function getNome() {
	    return $this->nome;
    }
	
    public function setNome($nome): self {
	    $this->nome = $nome;
	    return $this;
    }
	
    public function getConcluido() {
	    return $this->concluido;
    }
	
    public function setConcluido($concluido): self {
	    $this->concluido = $concluido;
	    return $this;
    }
	
    public function getPrioridade() {
        return $this->prioridade;
    }

    public function setPrioridade($prioridade) {
        $this->prioridade = $prioridade;
    }
}

?>
